app/controllers/articles_controller.rb

```diff
@@ -8,32 +8,28 @@ class ArticlesController < ApplicationController
 
   def feed
     skip_authorization
-    @page = params[:page].to_i
+
+    @articles = Article.where(published: true).
+      select(:published_at, :processed_html, :user_id, :organization_id, :title, :path).
+      order(published_at: :desc).
+      page(params[:page].to_i).per(12)
+
     if params[:username]
       if @user = User.find_by_username(params[:username])
-        @articles = Article.where(published: true, user_id: @user.id).
-          select(:published_at, :processed_html, :user_id, :organization_id, :title, :path).
-          order("published_at DESC").
-          page(@page).per(12)
+        @articles.where(user_id: @user.id)
       elsif @user = Organization.find_by_slug(params[:username])
-        @articles = Article.where(published: true, organization_id: @user.id).
-          includes(:user).
-          select(:published_at, :processed_html, :user_id, :organization_id, :title, :path).
-          order("published_at DESC").
-          page(@page).per(12)
+        @articles.where(user_id: @organization.id).includes(:user)
```

Comment:
Based on the failing spec, I think this is the change you need.
```suggestion
        @articles.where(organization_id: @user.id).includes(:user)
```

Reply:
Oh, shoot! I should stop coding at night :D